sale_order_product_picker/models/sale_order.py

```diff
@@ -60,12 +60,23 @@ def _list_product_picker_filters(self):
     def _get_partner_picker_field(self):
         # HACK: To avoid installation error when get default value to be used in a
         # depends of a computed field
-        if "sale_order_product_picker" in self.env.registry._init_modules and self.env[
-            "ir.default"
-        ].get("sale.order", "use_delivery_address"):
-            return "partner_shipping_id"
-        else:
-            return "partner_id"
+        if "sale_order_product_picker" in self.env.registry._init_modules:
```

Comment:
My question here:

The use of ir.default is questionning for me as it will not indicate which field is used through all sale orders... as this is a default for new ones.

So, we can have a mix through all sale orders.

IMHO, the implementation should be changed.